app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord # rubocop:disable Metrics/ClassLength
   has_one_time_password
 
   mount_base64_uploader :avatar, AvatarUploader
-  has_paper_trail skip: [:avatar]
+  has_paper_trail skip: %i[avatar cover_photo image], unless: proc { |o| o.archived? }
```

Comment:
Een user heeft geen `cover_photo` of `image`

Reply:
Ah jij had dit in `application_record` willen doen?

Reply:
Ja, paper trail maakt nu steeds 2 changes aan doordat we 2x has_paper_trail zeggen. Jij een mooie manier om dat op te lossen?

Reply:
Het expliciet doen van `has_paper_trail` in de modellen zelf, en niet in `application_record`. Want PaperTrail heeft niet een manier van overriden, zoals we daar nu achter komen :)

Maar de cover_photo en image moet hier nog steeds wel weg denk ik